app/src/main/java/com/android/partagix/model/Database.kt

```diff
@@ -388,6 +391,117 @@ class Database(database: FirebaseFirestore = Firebase.firestore) {
     onSuccess(user)
   }
 
+  /**
+   * Retrieve all comments that a user has received, both as a owner than as an loaner
+   *
+   * @param userId the user's id
+   * @param onSuccess the function to return a List containing pairs (comment's author name,
```

Comment:
it's not a function to return a list but the function that will be called with the resulting list when the db will give it.

Reply:
good